contracts/remarkables/src/test_utils.rs

```diff
@@ -56,3 +70,81 @@ impl CW721TestContract {
         Box::new(contract)
     }
 }
+
+pub const ADMIN: &str = "cosmos17qcf9sv5yk0ly5vt3ztev70nwf6c5sprkwfh8t";
```

Comment:
It defines in the desmos bindings modules mocks lib but it is not exposed. We should expose them.